gem/settings/base.py

```diff
@@ -37,31 +37,31 @@
 # Authentication Service Tokens
 USE_OIDC_AUTHENTICATION = environ.get(
     'USE_OIDC_AUTHENTICATION', '') == 'true'
-OIDC_RP_CLIENT_ID = "unused"  # Some constructors require that this be set.
-OIDC_RP_CLIENT_SECRET = "unused"  # some constructors require that this be set.
+OIDC_RP_CLIENT_ID = "118274"  # Some constructors require that this be set.
+OIDC_RP_CLIENT_SECRET = "2648264932dd8c5b171b54cf22f94e8d9cae30db5efc0eb59a0885bb"  # some constructors require that this be set.
 # <URL of the OIDC OP authorization endpoint>
 OIDC_OP_AUTHORIZATION_ENDPOINT = environ.get(
-    'OIDC_OP_AUTHORIZATION_ENDPOINT', '')
+    'OIDC_OP_AUTHORIZATION_ENDPOINT', 'https://authentication-service.qa-hub.ie.gehosting.org/openid/authorize/')
 # <URL of the OIDC OP token endpoint>
-OIDC_OP_TOKEN_ENDPOINT = environ.get('OIDC_OP_TOKEN_ENDPOINT', '')
+OIDC_OP_TOKEN_ENDPOINT = environ.get('OIDC_OP_TOKEN_ENDPOINT', 'https://authentication-service.qa-hub.ie.gehosting.org/openid/token/')
 # <URL of the OIDC OP userinfo endpoint>
-OIDC_OP_USER_ENDPOINT = environ.get('OIDC_OP_USER_ENDPOINT', '')
+OIDC_OP_USER_ENDPOINT = environ.get('OIDC_OP_USER_ENDPOINT', 'https://authentication-service.qa-hub.ie.gehosting.org/openid/userinfo/')
 OIDC_RP_SCOPES = 'openid profile email address phone site roles'
 OIDC_STORE_ID_TOKEN = True
-OIDC_OP = environ.get('OIDC_OP', '')
-THEME = environ.get('THEME', 'springster')
-LOGIN_REDIRECT_URL = environ.get('LOGIN_REDIRECT_URL', 'wagtailadmin_home')
+OIDC_OP = environ.get('OIDC_OP', 'https://authentication-service.qa-hub.ie.gehosting.org')
+THEME = environ.get('THEME', 'chhaa-jaa')
+LOGIN_REDIRECT_URL = environ.get('LOGIN_REDIRECT_URL', 'https://chajaa.redmatter.tech/')
 LOGIN_URL = 'molo.profiles:auth_login'
 LOGOUT_URL = 'molo.profiles:auth_logout'
 REGISTRATION_URL = reverse_lazy('molo.profiles:user_register')
 VIEW_PROFILE_URL = reverse_lazy('molo.profiles:view_my_profile')
 EDIT_PROFILE_URL = reverse_lazy('edit_my_profile')
-LOGOUT_REDIRECT_URL = environ.get('LOGOUT_REDIRECT_URL', '')
+LOGOUT_REDIRECT_URL = environ.get('LOGOUT_REDIRECT_URL', 'https://chajaa.redmatter.tech/')
```

Comment:
all of these settings are for you to use locally, I have updated the spinnaker config to reflect these as environment variables, but chajaa.redmatter.tech is your local instance and not the actual QA site